utopia-remix/app/handlers/listProjects.ts

```diff
@@ -0,0 +1,35 @@
+import { listProjects } from "../models/project.server";
```

Comment:
Definitely not using our regular Prettier formatting as we bin off the semicolons.